lib/agent/plugins/control-panel/interval/test/interval_spec.js

```diff
@@ -0,0 +1,193 @@
+var assert = require("assert"),
+    sinon  = require('sinon'),
+    should = require('should'),
+    api = require('../../api'),
+    interval = require(".."),
+    bus = require("../../bus"),
+    hooks = require('../../../../hooks'),
+    common = require('../../../../../common'),
```

Comment:
You're right. I was initially trying to make the test work with `common` and then I found in another test file how to fake the object.